src/reference/modules/administration/pages/procedures.adoc

```diff
@@ -182,7 +179,7 @@ Here is the restoration procedure:
 rudder agent disable
 
 # Stop Rudder services
-systemctl stop rudder-agent rudder-server
+systemctl stop rudder-agent rudder-server rudder-relayd
```

Comment:
rudder-server is an umbrella service that should include rudder-relayd.

If we don't want to use it it should be rudder-jetty instead.